cmd/dcrdata/public/scss/navigation.scss

```diff
@@ -38,6 +38,10 @@
   color: inherit;
 }
 
+.nav-link {
+  cursor: pointer;
+}
```

Comment:
Bad?  I tried `$nav-link-cursor: pointer;` in _variables.scss as suggested by the comment in bootstrap_overrides.scss, but that didn't work.

Reply:
Good.